src/api/procedures/modifyInstructionAffirmation.ts

```diff
@@ -52,8 +55,12 @@ export async function prepareModifyInstructionAffirmation(
   const { operation, id } = args;
 
   const instruction = new Instruction({ id }, context);
+  const { venue } = await instruction.details();
 
-  await assertInstructionValid(instruction, context);
+  await Promise.all([
+    assertVenueExists(venue.id, context),
```

Comment:
This is not necessary. Venues can't be removed, so if an instruction was created on one (as reflected by the fact that instruction.details returns it), then the Venue must exist. What I meant to say in my comment before was that we should validate whether the instruction exists (not knowing if it was already being done). Since it is already being done by `assertInstructionValid`, then we shouldn't do anything else